.rhcicd/clowdapp-backend.yaml

```diff
@@ -207,6 +207,11 @@ objects:
           value: ${QUARKUS_REST_CLIENT_KC_URL}
         - name: NOTIFICATIONS_USE_SOURCES_SECRETS_BACKEND
           value: ${SOURCES_ENABLED}
+        - name: SOURCES_PSK
+          valueFrom:
+            secretKeyRef:
+              name: sources-psk
+              key: psk
```

Comment:
I have named the secret like this because this might be useful in ephemeral, because Sources creates a Secret of the same name:

https://github.com/RedHatInsights/sources-api-go/blob/main/deploy/clowdapp.yaml#L5-L15

By naming the secret like this, Notifications can simply pick it up from the one that Sources creates. Although we can also use another name and key combination for the secret and simply write the PSK that Sources defined in their ClowdApp.

What do you think?

Reply:
Your link shows a slightly different name: `internal-psk`. Did I miss something or do we need to rename our secret?

Reply:
I am really surprised that I didn't even manage to copy and paste properly. You're absolutely right, its name is `internal-psk`.